fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib.yaml

```diff
@@ -0,0 +1,41 @@
+id: fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib
+branch: 1-19.08
+runtime: fr.free.hxc2001.HxCFloppyEmulator
+runtime-version: stable
+sdk: org.freedesktop.Sdk//19.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: ${FLATPAK_DEST}
+modules:
+  - name: capslib
+    buildsystem: simple
+    build-commands: 
+      - install -D apply_extra "${FLATPAK_DEST}/bin/apply_extra"
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib.metainfo.xml
+      - appstream-compose --basename=fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib --prefix=${FLATPAK_DEST} --origin=flatpak fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib
+    sources: 
+      - type: extra-data
+        only-arches: [x86_64]
+        url: http://www.softpres.org/_media/files:ipflib42_linux-x86_64.tar.gz
+        sha256: ea139673e29a36dc3be7ba916628b5c8a03353bd68be4d346daed803d472ddf3
+        size: 48272
+        filename: ipflib42_linux.tar.gz
+      - type: extra-data
```

Comment:
I don't think this is needed

Reply:
extra-data? I don't understand the license well enough to believe otherwise.

Reply:
I meant the i386 extra-data